x/interchainqueries/keeper/msg_server.go

```diff
@@ -77,6 +77,69 @@ func (k msgServer) RegisterInterchainQuery(goCtx context.Context, msg *types.Msg
 	return &types.MsgRegisterInterchainQueryResponse{Id: lastID}, nil
 }
 
+func (k msgServer) RemoveInterchainQuery(goCtx context.Context, msg *types.MsgRemoveInterchainQueryRequest) (*types.MsgRemoveInterchainQueryResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+	ctx.Logger().Debug("RemoveInterchainQuery", "msg", msg)
+
+	if err := msg.ValidateBasic(); err != nil {
```

Comment:
Don't we already call `ValidateBasic` in message bindings? Seems like code duplication for me.

Reply:
You are right. It will be fixed in separate task.

Reply:
Do we already have a task in backlog for it?

Reply:
I'll create it